ui/imports/shared/popups/send/views/AmountToSend.qml

```diff
@@ -142,8 +142,6 @@ ColumnLayout {
             input.edit.color: input.valid ? Theme.palette.directColor1
                                           : Theme.palette.dangerColor1
             input.edit.readOnly: !root.interactive
-            validationMode: StatusInput.ValidationMode.OnlyWhenDirty
```

Comment:
I wonder how come this is crashing; also is there an issue logged for this?

Reply:
Oh I see... you introduced a duplicate binding

Reply:
I also wonder how this breaking change ended up being master since this UI QML test is clearly failing on it:
```
[2024-06-10T11:03:42.364Z] 3: QWARN  : QmlTests::UnknownTestFunc() file:///home/jenkins/workspace/s_linux_x86_64_tests-ui_PR-15115/storybook/qmlTests/tests/tst_SwapInputPanel.qml:55:5: Type SwapInputPanel unavailable 
[2024-06-10T11:03:42.364Z] 3:          property SwapInputPanel controlUnderTest: null 
[2024-06-10T11:03:42.364Z] 3:          ^
[2024-06-10T11:03:42.364Z] 3: QWARN  : QmlTests::UnknownTestFunc() file:///home/jenkins/workspace/s_linux_x86_64_tests-ui_PR-15115/ui/app/AppLayouts/Wallet/panels/SwapInputPanel.qml:183:13: Type AmountToSend unavailable 
[2024-06-10T11:03:42.364Z] 3:                  AmountToSend { 
[2024-06-10T11:03:42.364Z] 3:                  ^
[2024-06-10T11:03:42.364Z] 3: QWARN  : QmlTests::UnknownTestFunc() file:///home/jenkins/workspace/s_linux_x86_64_tests-ui_PR-15115/ui/imports/shared/popups/send/views/AmountToSend.qml:143:13: Property value set multiple times 
[2024-06-10T11:03:42.364Z] 3:                  validationMode: StatusInput.ValidationMode.Always 
[2024-06-10T11:03:42.364Z] 3:                  ^
[2024-06-10T11:03:42.364Z] 3/3 Test #3: QmlTests .........................***Failed   12.70 sec
```

Reply:
Yeah that can happen but the latter change (whatever it was) should have been stopped by the failing test right?

Reply:
OK, I get it now, if the two (semantically conflicting) PRs hadn't actually created a (git) merge conflict, then we end up being in this broken state :/